wandb/sdk/wandb_artifacts.py

```diff
@@ -192,6 +195,11 @@ def __init__(
             self._incremental = incremental
             wandb.termwarn("Using experimental arg `incremental`")
 
+        self._disable_cache = False
```

Comment:
Walking through everything that happens between L174 and here I can't tell why you're setting it back to false. What's going on that could change it? Or is it for the sake of clarity, and you want to highlight that `_disable_cache` is False when `get_artifacts_cache()` gets called regardless of the environment variable?

Reply:
ops thats a missed typo